packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -294,7 +294,10 @@ export class TokenDetectionController extends BaseController<
         }
 
         if (tokensToAdd.length) {
-          await this.addDetectedTokens(tokensToAdd);
+          await this.addDetectedTokens(tokensToAdd, {
+            selectedAddress,
+            chainId,
```

Comment:
Is this a type mismatch?  Does it need to be `{ chainId }`?  Per the method in TokenControllers.js, this is supposed to be an object, but it appears you're passing a string.

Reply:
It should be an object containing a `selectedAddress` and `chainId`: https://github.com/MetaMask/controllers/pull/956/files#diff-4de3568278666a6d7a1c5084b38802cc730ba3ce94b7fd90de875d758e462d99R450

Reply:
My apologies, I read this wrong the first time!